openstack-helm/trilio-backup-targets/values_overrides/other_s3.yaml

```diff
@@ -0,0 +1,28 @@
+trilio_backup_target:
+  backup_target_name: 'S3_BackupTarget'
+  backup_target_type: 's3'
+  is_default: False
+  s3_type: 'other_s3'
+  s3_access_key: 'ACCESSKEY1'
+  s3_secret_key: 'SECRETKEY1'
+  s3_region_name: 'REGION1'
+  s3_bucket: 'BUCKET1'
+  s3_endpoint_url: 'ENDPOINT_URL1'
+  s3_signature_version: 'default'
+  s3_auth_version: 'DEFAULT'
+  s3_bucket_object_lock_enabled: false
+  s3_ssl_enabled: true
+  s3_ssl_verify: true
+  s3_self_signed_cert: true
+  s3_ssl_ca_cert: |
+    SSL_CERT_LINE_1
+    SSL_CERT_LINE_2
+    SSL_CERT_LINE_N
+
+common:
+  vault_data_dir: /var/lib/trilio/triliovault-mounts
+  nova_user_id: "42424"
+  nova_group_id: "42424"
+
+images:
+  trilio_backup_targets: docker.io/trilio/trilio-wlm-helm:shruti-6.1.0-bobcat-2
```

Comment:
We need to use image nmae as trilio_wlm instead of trilio_backup_targets.
User should clearly know that is this wlm image.